File: Entities/Medication.cs
namespace PrivateHospitalSystem.Entities
{
    public class Medication
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty; 
        public int StockQuantity { get; set; }
        public int MinimumStockAlert { get; set; } = 10; 

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: frontend/src/types/medication.ts
export interface Medication {
  id: string
  name: string
  stockQuantity: number
  minimumStockAlert: number
  isLowStock: boolean
  batchNumber?: string
  expiryDate?: string
}

export interface CreateMedicationDto {
  name: string
  stockQuantity: number
  minimumStockAlert: number
  batchNumber?: string
  expiryDate?: string
}